tests/test_fakes.py

```diff
@@ -1522,3 +1522,24 @@ def test_json_extract_cast_as_varchar(dcur: snowflake.connector.cursor.DictCurso
 
     dcur.execute("SELECT j:str::number as j_str_number, j:num::number as j_num_number FROM example")
     assert dcur.fetchall() == [{"J_STR_NUMBER": 100, "J_NUM_NUMBER": 200}]
+
+
+def test_truncate(dcur: snowflake.connector.cursor.DictCursor):
+    dcur.execute("CREATE TABLE example (i INTEGER)")
+    dcur.execute("""INSERT INTO example VALUES (1)""")
+
+    dcur.execute("TRUNCATE TABLE example")
+    assert dcur.description == [
+        ResultMetadata(
+            name="status",
+            type_code=2,
+            display_size=None,
+            internal_size=16777216,
+            precision=None,
+            scale=None,
+            is_nullable=True,
+        )
+    ]
```

Comment:
To match the snowflake result set this would need to assert:
```suggestion
    assert dcur.fetchall() == [{"status": "Statement executed successfully."}]
```

To do this, try adding another branch [here](https://github.com/dancardin/fakesnow/blob/9ecb6dca5b632c3feced47cc65e8c5e070a3e56d/fakesnow/cursor.py#L318) which sets `result_sql = SQL_SUCCESS`.

You won't need to change `_describe_last_sql`